Gopkg.lock

```diff
@@ -9,6 +9,14 @@
   revision = "d76b18b42f285b792bf985118980ce9eacea9d10"
   version = "v1.3.0"
 
+[[projects]]
```

Comment:
Wouldn't be the proper PR for this, but should be locked down anyway, so might as well take it here.